README.md

```diff
@@ -47,14 +45,12 @@ def send_appropriate_email(input:)
 end
 ```
 
-This re-expression preserves all the original behavior, but accentuates a few
-painful things:
+This re-expression preserves original behavior, accentuating two pain points:
 
 1. What happens when `subject` is nil?
```

Comment:
It doesn't really, other than it sends the `nil` as argument to the `send_email` method as `subject` keyword arg. Perhaps articulate that more clearly above.